README.md

```diff
@@ -68,6 +46,66 @@ Where any public assets are stored.
 #### `scripts`
 Tooling and scripts to make the repository run smoothly and correctly.
 
+## Development
+
+### Starting the application locally
+##### Prerequisite steps
+1. Install [node.js](https://nodejs.org/en/download/package-manager). For example, installing version 20 using brew:
+    ```bash
+    brew install node@20
+    ```
+1. Clone this repo to your local workspace. For example, using github cli:
+    ```bash
+    gh repo clone DSACMS/iv-verify
+    ```
+    
+##### Steps
+1. Change into the repo's directory.
+    ```bash
+    cd iv-verify
+    ```
+1. Install dependencies.
+    ```bash
+    npm install
+    ```
+1. Run the development server.
+    ```bash
+    npm run dev
+    ```
+1. Open [http://localhost:3000/](http://localhost:3000/) with your browser to access the application.
+
+### Running the application in a docker container
+
+##### Prerequisite steps
```

Comment:
for the `PUPPETEER` workaround, do you want to reference that here just in case people run into some unexpected failure state with it?

Reply:
Readme updated.